admin/schedule-form-excludes.php

```diff
@@ -303,24 +291,21 @@ class="dashicons dashicons-update"
 					</td>
 
 					<td>
-						<?php echo esc_html( substr( sprintf( '%o', $file->getPerms() ), - 4 ) ); ?>
+						<?php if ( ! $is_unreadable ) {
```

Comment:
This is the line which matters, we needed to check if the file was readable before calling `getPerms`